ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -101,17 +98,13 @@ object ParallelIndexerFactory {
                 // this is the DataSource which will be wrapped by HikariCP, and which will drive the ingestion
                 // therefore this needs to be configured with the connection-init-hook, what we get from HaCoordinator
                 dataSource = dataSourceStorageBackend.createDataSource(
-                  jdbcUrl = jdbcUrl,
-                  dataSourceConfig = dataSourceConfig,
+                  dataSourceConfig = dbConfig.dataSourceConfig,
                   connectionInitHook = Some(connectionInitializer.initialize),
                 ),
                 serverRole = ServerRole.Indexer,
-                connectionPoolSize =
-                  ingestionParallelism + 1, // + 1 for the tailing ledger_end updates
```

Comment:
I could not find the place where the connection pool size of the parallel indexer is tied to the ingestionParalellism parameter. Please note that the connection pool size is different for the indexer and the ledger api uses of the db

Reply:
In the new branch it is about defaults:

https://github.com/digital-asset/daml/blob/skisel-da/cli-config-layer/ledger/participant-integration-api/src/main/scala/platform/indexer/IndexerConfig.scala#L42

My idea was is to document this in the default configuration